Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM amazonlinux:2017.03
+
+MAINTAINER akrug@mozilla.com
+
+# Patch this image
+
+RUN yum update -y
+
+# Install python 3.6.1
+
+RUN yum install \
+    gcc wget findutils \
+    zlib zlib-devel openssl-devel \
+    libffi-devel git \
+    -y
+
+WORKDIR /usr/src/
+
+RUN wget https://www.python.org/ftp/python/3.6.1/Python-3.6.1.tgz
```

Comment:
as for the image, this makes it hard to update python since its manual, vs  using a distro package

Reply:
For serverless framework deployment only.  Not for use in the lambda sandbox.